### DISCUSSION TITLE
BPF_XDP_EGRESS support

### DISCUSSION THREAD

**Discussion by boratanrikulu** [cilium/ebpf#911] [Q&A] [answered]
Is `BPF_XDP_EGRESS` supported in current ebpf-go?
https://lwn.net/Articles/818591/  
https://patchwork.ozlabs.org/project/netdev/patch/20200123014210.38412-3-dsahern@kernel.org/

**Comment by florianl** [comment, marked as answer]
The patch for `BPF_XDP_EGRESS` was not accepted and therefore is not part of the Linux kernel.
